CoreFramework/DriverCore/WebDriverAction.cs

```diff
@@ -448,6 +448,32 @@ public void AssertEquals(object actual, object expected)
             throw excep;
         }
     }
+    public void AssertListAscending(List<string> list)
+    {
+        try
+        {
+            list.Should().BeInAscendingOrder();
+            HtmlReport.Pass("List is sorted in ascending order");
+        }
+        catch(Exception excep)
+        {
+            HtmlReport.Fail("List is not sorted in ascending order");
```

Comment:
print the list of strings.
If fail, you can know exact current order

Reply:
Done